client/js/lounge.js

```diff
@@ -133,6 +133,14 @@ $(function() {
 					data: target.data("id"),
 				});
 			}
+			if (target.hasClass("channel") && utils.hasUserModeInChannel($(`#chan-${target.data("id")}`))) {
```

Comment:
I'm not 100% on when to show this item.

On the one hand, on all the irc servers I use, any user can view the banlist, so should we show it for everyone? On the otherhand, I thought that only people with some sort of privileges in the channel would have a **reason** to view it, so we should probably only show it for them.

Open to suggestions.

Reply:
Yeah I see no reason to limit this to user modes, just display it to everyone.

Reply:
It might be nice to not show it for regular people as makes the functionality more accessible where people now might not be aware of it being possible unless they run a channel. It might result in some soapboxing in some channels if people are made more aware of the banlist. 

Technically there is no reason not to show it indeed.

Reply:
Yeah, my reasoning is 2-fold:

1. It makes the context list busier when there's probably no reason for you to be calling the banlist if you aren't at least voiced
1. Making it more accessible for users can lead to them unnecessarily calling it, and could lead to issues as explained by @creesch 

If people think we should just show it for everything, that's fine. @astorije , what do you think with your UX hat on?

Reply:
I can see both sides of the argument, but I am in favor of displaying it for normal users. I'll try to list my reasons here (don't see that as patronizing, I'm just too tired to gather thoughts properly and listing helps me compartimentalize :D).

- Users can still send the `/banlist` command, so it's not like we're completely blocking the feature (though we could, but I don't think we should), we're just hiding it.
- Adding a UI element for it helps with discoverability, which is nice.
- The context menu becoming busier is a valid point, but at the moment we only have 1 real entry in there (2 if you count the entry that clicks on the target :) ) so I don't see that as an issue for now. We can for sure re-think that when it becomes a problem, and there are paradigms for that, such as grouping entries, or re-evaluate what we display in there.
- You rightfully mention that this is primarily useful for "power users" or actually "voiced or op'ed users", which is a good reason why this shouldn't be in the main UI (i.e. constantly on the page, like #1836 adds). But this is in a context menu, which is a good place where to put advanced features without burying them completely, I think.
- Though it's clearly useful for voiced/op users, I can see how it's useful for regular users. Just like the question someone asked today "why would someone connect to the same network multiple times" (which is even less obvious IMO ahah), I'm sure there are plenty use cases for not being voiced on a channel you actively contribute/operate, or checking banlists on channels you don't operate, etc. In fact, I was working in a (private) team where no one was ever op or voiced to leave everyone on the same hierarchy level. Workflows :man_shrugging: 
- Re: soapboxing (thanks, I learned a new word :D), I get what you're saying, but I don't think it's our business at least in this case. If we were a SaaS solution or something proprietary like that, I'd probably have a different point of view.

So yeah, I'd just display it :)

Reply:
Alrighty, you've convinced me, I'll enable it for everyone. 

Should I remove the changes I made to the op function, or should I put that in a different PR?

Reply:
I have removed the changes to the op function and put them in #1864